### ISSUE TITLE
Updating profile removes a user from member directory page

### ISSUE THREAD

**Issue by jacobarriola** [ultimatemember/ultimatemember#165]
Repro steps:
Edit your profile (ie change first name)

Expected results:
Profile changes 
Profile still shows in member directory

Actual results:
Profile field value updates
Profile no longer shows up in member directory

We noticed this is related to how you filter out members who do not want to be listed.  In `um_remove_special_users_from_list()` from `um_filters-members.php` line 49, it seems as though you are using `NOT EXISTS` in your query even though a `hide_in_members` will exists with a yes or no value.

To temporarily fix, we're hooking into your `um_after_user_account_updated` action to remove the key entirely from a user's meta.

```php
add_action( 'um_after_user_account_updated', function( $um_user_id ) {

  if ( empty( $_REQUEST['hide_in_members'] ) ) {
    return;
  }

  if ( $_REQUEST['hide_in_members'] === ['No'] ) {
    delete_user_meta( $um_user_id, 'hide_in_members' );
  }

} );
```

**Comment by champsupertramp** [comment]
Hi @jacobarriola 

Are you using the Latest Version on the WP repo or the pre-released version in the link?
https://github.com/ultimatemember/ultimatemember/releases

Regards,

**Comment by jacobarriola** [comment]
I'm on 1.3.68.

**Comment by champsupertramp** [comment]
Hi @jacobarriola , Can you please try the pre-released version 1.3.69 in the link?
https://github.com/ultimatemember/ultimatemember/releases

It has fixes to the missing profiles in Member Directory.

Regards,

**Comment by jacobarriola** [comment]
Thx. Fixed with latest release. 💯
